assembl/scripts/ini_files.py

```diff
@@ -380,6 +438,9 @@ def diff_ini(first, second, diff=None, existing_only=False):
 def compose(rc_filename, random_file=None):
     """Compose local.ini from the given .rc file"""
     rc_info = combine_rc(rc_filename)
+    rc_info['*code_root'] = code_root(rc_info)
+    # Special case: uwsgi does not do internal computations.
+    rc_info['uwsgi__virtualenv'] = venv_path(rc_info)
```

Comment:
USWGI does handoff to the virtualenv to get the proper version of the files?

Reply:
yes. BUT no variable substitution in that case, as per comment.